### DISCUSSION TITLE
what is the difference between surface_Delaunay_remeshing() and isotropic_remeshing()

### DISCUSSION THREAD

**Discussion by citystrawman** [CGAL/cgal#8434] [Q&A] [answered]
_Please use the following template to help us solving your issue._

## Issue Details

I am not clear what is the difference between `surface_Delaunay_remeshing()` and `isotropic_remeshing()`
As I understand, the goal of` isotropic_remeshing()` is to create a mesh where all the edges have approximately the same length, and the triangles are as equilateral as possible. While  `surface_Delaunay_remeshing()` aims to produce a mesh that satisfies the Delaunay criterion on a surface. This generally means that for any triangle in the mesh, the circumcircle of the triangle does not contain any other vertex of the mesh.

By intuition, if a surface is isotropic remeshed, then it should satisfies the Delaunay criterion because all triangles are almost equilateral. So why CGAL differentiate the two methods? (I think I may not have a full understanding of Delaunay criterion )

## Source Code

_If your issue arises by using CGAL in your own source code, please provide a minimalist example that we can compile easily to reproduce the bug. If your issue arises from using a CGAL program (demo, example, etc.), please let us know which one. Helping you solving an issue is much easier and efficient if we can reproduce it._

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):
* Compiler:
* Release or debug mode:
* Specific flags used (if any):
* CGAL version:
* Boost version:
* Other libraries versions if used (Eigen, TBB, etc.):

**Comment by janetournois** [comment, marked as answer]
`surface_Delaunay_remeshing()` offers a lot more meshing parameters, such as approximation distance to the input and an angle bound on triangles. This method relies on the 3D tetrahedral mesh generation engine.

`isotropic_remeshing()` is a lighter algorithm, designed to remesh triangulated surfaces, with only an edge length criterion. 

More details can be found in the [user manual](https://doc.cgal.org/latest/Polygon_mesh_processing/index.html#Remeshing)

**Comment by citystrawman** [comment]
> `surface_Delaunay_remeshing()` offers a lot more meshing parameters, such as approximation distance to the input and an angle bound on triangles. This method relies on the 3D tetrahedral mesh generation engine.
> 
> `isotropic_remeshing()` is a lighter algorithm, designed to remesh triangulated surfaces, with only an edge length criterion.
> 
> More details can be found in the [user manual](https://doc.cgal.org/latest/Polygon_mesh_processing/index.html#Remeshing)

So is it possible tha by setting the parameters in `surface_Delaunay_remeshing()` (such as mesh_edge_size, mesh_facet_angle), I can achieve the same result like `isotropic_remeshing()`?

**Comment by janetournois** [comment]
I guess that a smart combination of parameters, on not-too-curved surfaces, could reach a similar result.
However, imho it would be better to look for what criteria you need to be satisfied for your output mesh, and chose the meshing algorithm that is better at reaching them.
